internal/pkg/cli/pipeline_init.go

```diff
@@ -205,21 +210,19 @@ func (o *initPipelineOpts) Execute() error {
 	return nil
 }
 
-// RecommendedActions returns follow-up actions the user can take after successfully executing the command.
-func (o *initPipelineOpts) RecommendedActions() []string {
+// RequiredActions returns follow-up actions the user can take after successfully executing the command.
```

Comment:
Hmm why do we stop recommending?

Reply:
Because they're more urgent than recommendations--they're required in order for pipelines to actually build.